src/modules/tracker.js

```diff
@@ -2235,6 +2235,390 @@ class Tracker {
     return new Error('parameters are required of type object');
   }
 
+  /**
+   * Send ASA request submitted event
+   *
+   * @function trackAssistantSubmit
+   * @param {object} parameters - Additional parameters to be sent with request
+   * @param {string} parameters.intent - Intent of user request
+   * @param {string} [parameters.section] - The section name for the item Ex. "Products"
+   * @param {object} [networkParameters] - Parameters relevant to the network request
+   * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
+   * @returns {(true|Error)}
+   * @description User submitted an assistant search
+   *   (pressing enter within assistant input element, or clicking assistant submit element)
+   * @example
+   * constructorio.tracker.trackAssistantSubmit(
+   *     {
+   *         intent: 'show me a recipe for a cookie',
+   *     },
+   * );
+   */
+  trackAssistantSubmit(parameters, networkParameters = {}) {
+    if (parameters && typeof parameters === 'object' && !Array.isArray(parameters)) {
+      // Ensure parameters are provided (required)
+      const baseUrl = `${this.behavioralV2Url}assistant_submit?`;
+      const {
+        section,
+        intent,
+      } = parameters;
+      const queryParams = {};
+      const bodyParams = {
+        intent,
+        section,
+      };
+
+      if (section) {
+        queryParams.section = section;
+      }
```

Comment:
I noticed that this was the only tracking function that applied `section` to the query params. Is that expected?

Reply:
Nope, removed it.